components/primary_button.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { StyleSheet, Text, View, Button } from 'react-native';
+
+
+
+export default class primary_button extends React.Component {
+  _onPressButton() {
+    Alert.alert('You tapped the button!')
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+         <View style={styles.RectangleShapeViewOne}></View>
+         <View style={styles.buttonContainer}>
+          <Button
+            onPress={this._onPressButton}
+            title="Press Me"
+            color = '#fff'
+          
+          />
+            </View>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  RectangleShapeViewOne: {
```

Comment:
Don't include numbers in variables, it's too specific of a style name
Als use camel case not Pascal - for example instead of `ThisIsIt` it should be `thisIsIt`